packages/joint-core/docs/src/joint/api/dia/CellView/prototype/findBySelector.html

```diff
@@ -1,12 +1,9 @@
-<pre class="docs-method-signature"><code>cellView.findBySelector([selector, root, selectors])</code></pre>
-<p>Return an array of subelements of this CellView which are identified by <code>selector</code>. If no such subelements are found, return an empty array.</p>
+<pre class="docs-method-signature"><code>cellView.findBySelector(selector)</code></pre>
```

Comment:
We are changing the method signature here but not in the code - shouldn't we acknowledge the `root` and `selectors` options here but mark them as deprecated?